### PR TITLE
Create Navbar component

### PR DESCRIPTION
**Pull request by salih18**
Created navbar component.

### CODE REVIEW

File: client/src/pages/layout/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Navbar as NavbarB, NavDropdown, Nav } from "react-bootstrap";
```
Comment (sarah-vanderlaan) [inline client/src/pages/layout/Navbar.js:3]:
```suggestion
import { Navbar as NavbarBootstrap, NavDropdown, Nav } from "react-bootstrap";
```

lets follow Frank's advice from your last PR to give components really descriptive names - makes it more easy for future developers to read and understand your code

File: client/src/pages/layout/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      <NavbarB.Toggle aria-controls="basic-navbarB-nav" />
+      <NavbarB.Collapse id="basic-navbarB-nav">
+        <Nav className="ml-auto">
+          <NavDropdown
+            className="navbar-dropdown"
+            title="HOME"
+            id="navbar-dropdown"
+          >
+            <NavDropdown.Item as={Link} to="/">
+              About
```
Comment (sarah-vanderlaan) [inline client/src/pages/layout/Navbar.js:20]: Have you defined pages for About/Faqs/Contact? If you want these pages - lets make tickets for them in the project board.  Maybe it is better to remove these for now and then add them in when we have the pages implemented

File: client/src/pages/layout/Navbar.js (hunk 1 above)
Comment (salih18) [reply client/src/pages/layout/Navbar.js:3]: Yeah. I have just remembered after pushed it. Frank made a same feedback. Thanks

File: client/src/pages/layout/Navbar.js (hunk 2 above)
Comment (salih18) [reply client/src/pages/layout/Navbar.js:20]: Actually we have discussed About page with Ammar. Yes I can remove them for now.

**Review by sarah-vanderlaan — APPROVED** [review]

**Pull request merged**